WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-the-AD-FS-Configuration-Database.md

```diff
@@ -9,8 +9,10 @@ ms.topic: article
 ---
 
 
+
+
```

Comment:
Strictly speaking, it is not necessary to use more than 1 blank line (or 2, in case of frequent modifications), so I recommend removing these blank lines:
```suggestion
```